rhel/automake.mk

```diff
@@ -15,6 +15,7 @@ EXTRA_DIST += \
 	rhel/usr_lib_systemd_system_ovn-controller-vtep.service \
 	rhel/usr_lib_systemd_system_ovn-ic.service \
 	rhel/usr_lib_systemd_system_ovn-northd.service \
+	rhel/usr_lib_systemd_system_ovsdb-server-ic.service \
```

Comment:
The name "ovsdb-service-ic" doesn't tell the fact that this service is for OVN. Would it be better to be named as ovn-ic-db? (or ovsdb-ovn-ic)

Reply:
+1. I agree. ovn-ic-db seems better to me.

Reply:
No problem, I was waiting for naming proposal :)